infer/environment.cc

```diff
@@ -944,6 +944,13 @@ core::TypePtr Environment::processBinding(core::Context ctx, const cfg::CFG &inW
                 tp.type = std::move(argType);
                 tp.origins.emplace_back(core::Loc(ctx.file, bind.loc));
             },
+            [&](cfg::ArgPresent *i) {
+                /* Return an opaque boolean value that indicates whether or not arg was provided */
```

Comment:
```suggestion
                // Return an unanalyzable boolean value that indicates whether or not arg was provided
                // It's unanalyzable because it varies by each individual call site.
```